### PR TITLE
Allow users to upload files as attachments to text comments

### PR DESCRIPTION
**Pull request by adam-thomas**
@incuna/backend More comment attachment stuff.

This PR only adds the new form/view/URL and doesn't displace any of the existing functionality.

As with my other PRs from this week, I'll merge it once Travis is happy with it and I've checked it over; feel free to make comments after the fact.

### CODE REVIEW

**Comment by adam-thomas** [comment]
attachment form 1

attachment form 2

File: groups/forms.py [hunk 1]
```diff
@@ -54,6 +54,31 @@ class Meta:
...
 
 
+class AddTextCommentWithAttachment(BaseAddCommentForm):
+    file = forms.FileField()
+
+    def __init__(self, *args, **kwargs):
+        """Add nicer field labels."""
+        super(AddTextCommentWithAttachment, self).__init__(*args, **kwargs)
+        self.fields['body'].label = 'Comment'
+        self.fields['file'].label = 'Attachment'
```
Comment (meshy) [inline groups/forms.py]: You can define `labels` on the `class Meta`:

```python
class Meta:
   ...
    labels = {
        'body': Comment',
        'file': Attachment',
    }
```

Comment (adam-thomas) [inline groups/forms.py]: That is _so_ much better.  Thanks!

Comment (meshy) [inline groups/forms.py]: :+1:

**Comment by Ian-Foote** [comment]
The `Attachment` field in your screenshot has a `*` - I assume this means the field is required, but I'm not sure that's the behaviour we want.

**Comment by adam-thomas** [comment]
You can post comments without attachments elsewhere - this is the "upload a file to this discussion" page.

**Pull request merged**